### PR TITLE
NEPT-153: Behat parallelization

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    "drush/drush": "8.0.5",
     "ext-json": "*",
     "ext-phar": "*",
     "ext-xml": "*",
     "pear/versioncontrol_git": "dev-master",
     "pfrenssen/phpcs-pre-push": "1.0",
-    "phing/phing": "~2.10",
-    "php": ">=5.4.0",
-    "typhonius/behattask": "1.0"
+    "phing/phing": "~2.10"
```
Comment (drupol) [inline composer.json]: ~2 might be ok I think.

File: composer.json [hunk 2]
```diff
@@ -20,32 +20,8 @@
...
-        "source": {
-          "url": "https://gist.github.com/5719e15be9687ff7c138.git",
-          "type": "git",
-          "reference": "0ea666882cba3099dfa4feeb02e1bb85b2c0bde9"
-        },
-        "autoload": { "classmap": [ "BehatTask.php" ] },
-        "include-path": [ "." ]
-      }
+      "type": "vcs",
+      "url": "https://git.drupal.org/project/phingdrushtask.git"
```
Comment (drupol) [inline composer.json]: 👍

File: build.xml [hunk 3]
```diff
@@ -16,10 +16,10 @@
     <includepath classpath="src/Phing" />
 
     <echo msg="Loading Drush task." />
-    <taskdef name="drush" classname="DrushTask" />
+    <import file="${project.basedir}/vendor/drupal/phingdrushtask/import.xml"/>
 
-    <echo msg="Loading Behat task." />
-    <taskdef name="behat" classname="BehatTask" />
+    <echo msg="Loading Behat tasks." />
+    <import file="${project.basedir}/vendor/drupol/phingbehattask/import.xml"/>
```
Comment (drupol) [inline build.xml]: 👍

File: build.test.xml [hunk 4]
```diff
@@ -98,6 +98,16 @@
...
+    <!-- Set up Behat balancer. -->
+    <target name="setup-behat-balancer">
+        <behat:balancer
+            containers="${behat.load_balancer.containers}"
+            root="${behat.load_balancer.root}"
+            destination="${behat.load_balancer.destination}"
+            import="${behat.load_balancer.import}"
+        />
+    </target>
+
```
Comment (drupol) [inline build.test.xml]: 👍

File: composer.json (hunk 1 above)
Comment (ademarco) [inline composer.json]: I'd leave `"~2.10"` we don't really need latest version of Phing at the end

**Comment by voidtek** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Comment by drupol** [comment]
:+1:

**Comment by voidtek** [comment]
:+1:

**Pull request merged**